governance/default.go

```diff
@@ -1108,12 +1116,35 @@ func GetGovernanceItemsFromChainConfig(config *params.ChainConfig) GovernanceSet
 			params.CommitteeSize: istanbul.SubGroupSize,
 		}
 
-		for k, v := range istanbulMap {
-			if err := g.SetValue(k, v); err != nil {
-				writeFailLog(k, err)
-			}
+		appendToReturn(istanbulMap)
+	}
+
+	// magma params
+	if config.IsMagmaForkEnabled(common.Big0) &&
+		config.Governance.KIP71 != nil {
+		kip71 := config.Governance.KIP71
+		governanceMap := map[int]interface{}{
+			params.LowerBoundBaseFee:         kip71.LowerBoundBaseFee,
+			params.UpperBoundBaseFee:         kip71.UpperBoundBaseFee,
```

Comment:
LowerBoundBaseFee will be set from genesis.json, but UpperBoundBaseFee will be set as an empty value (0).
There's no check, so it will show an undefined behavior.